raiden/transfer/node.py

```diff
@@ -294,8 +294,7 @@ def subdispatch_targettask(
 
     elif sub_task and isinstance(sub_task, PaymentMappingState.TargetTask):
         is_valid_subtask = (
-            token_network_identifier == sub_task.token_network_identifier and
-            token_address == sub_task.token_address
+            token_network_identifier == sub_task.token_network_identifier
```

Comment:
Why did you remove the `token_address` check in the state machine here? What does this code have to do with eth-tester?

Reply:
that's a bug in Raiden - `token_address` is no longer part of the `TargetTask` structure, accessing it will cause an exception.

Reply:
Hmm I see, yes we missed that one in commit: 0bee56ffca77d72540d808b59806ded434ec013d

The initiator and mediator task had it removed. Can you also remove `token_address` from the function and from where it's called up the chain as it's not needed?